zkevm-circuits/src/copy_circuit.rs

```diff
@@ -343,15 +343,18 @@ impl<F: Field> SubCircuitConfig<F> for CopyCircuitConfig<F> {
                 Target::Memory.expr(),
                 meta.query_advice(id.lo(), Rotation::cur()), // call_id
                 meta.query_advice(addr, Rotation::cur()),    // memory address
-                0.expr(),
-                0.expr(),
-                meta.query_advice(value, Rotation::cur()),
-                0.expr(),
-                0.expr(),
-                0.expr(),
+                0.expr(),                                    // field tag
```

Comment:
Here the field_tag should be `None` in rw_table, since we are lookup .
https://github.com/privacy-scaling-explorations/zkevm-circuits/blob/main/zkevm-circuits/src/witness/rw.rs#L556. 

Also can see the condition to activate lookup
```
let cond = meta.query_fixed(q_enable, Rotation::cur())
                * tag.value_equals(CopyDataType::TxLog, Rotation::cur())(meta);
```
Those rows still meaning in and not fulfill the meaning of `CopyDataType::Padding`. I think keep `0.expr()` here is more appropriate

Reply:
Ah, my bad. I checked the memory read part on the rw table again, and the field tag corresponding to `5 FieldTag k3` is indeed empty.